src/core/Akka.Remote.Tests.MultiNode/RemoteQuarantinePiercingSpec.cs

```diff
@@ -0,0 +1,139 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RemoteQuarantinePiercingSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+
+using System;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.TestKit;
+
+namespace Akka.Remote.Tests.MultiNode
+{
+    public class RemoteQuarantinePiercingConfig : MultiNodeConfig
+    {
+        public RoleName First { get; }
+        public RoleName Second { get; }
+
+        public RemoteQuarantinePiercingConfig()
+        {
+            First = Role("first");
+            Second = Role("second");
+
+            CommonConfig = DebugConfig(false)
+                .WithFallback(ConfigurationFactory.ParseString(@"
+                  akka.loglevel = INFO
+                  akka.remote.log-remote-lifecycle-events = INFO
+                "));
```

Comment:
We have a lot of examples here in the code, when we are suing INFO/on/off with this setting